model/histogram/float_histogram.go

```diff
@@ -334,6 +334,33 @@ func (h *FloatHistogram) Equals(h2 *FloatHistogram) bool {
 	return true
 }
 
+// Size returns the size of the whole fields histogram in bytes.
```

Comment:
This should make clear that it includes the pointer to the histogram.

How about "Size returns the total size of the FloatHistogram, which includes the size of the pointer to FloatHistogram, all its fields, and all elements contained in slices."